tests/jenkins/groupvars.yml

```diff
@@ -1,163 +0,0 @@
----
-# dataverse/defaults/main.yml
```

Comment:
I was having to submit a PR and wait on the review process each time someone requested a new feature and accompanying group_var in dataverse-ansible. Since this file only applied beneath the dataverse-ansible role, I moved it there. Let me know if this should be removed in a separate issue.

Reply:
No, it's fine. Thanks.